### DISCUSSION TITLE
Flac Stream Offline

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4695] [Q&A]
Hi, I created a new station with .flac files and chose .flac format in the mount point. I updated the server to the latest rolling release and re-started the station, but the stream still shows as offline. Do flac files require any other set up to stream?

**Comment by ivellios1988** [comment]
Are you sure everything is OK with your liquidsoap.liq? Are there any error messages in logs?

**Comment by BeyondChrispy** [comment]
I am also finding that any stations that stream using FLAC show as offline even though they are working fine (although my stations don't use AutoDJ, the BUTT encoder connects directly to the mount).  This occurs on both my AzuraCast instances running v0.14.1 Stable • Docker • PHP 8.0.

**Comment by Citizen3rased** [comment]
I'm also have this same issue when trying to stream .flac files using a flac mountpoint. The stream shows 'Station offline" on the manage and public pages, even though the stream is actually playing.

**Comment by Piratenkanon** [comment]
i have the same, but is only shown, the stream playing wel, show here [(https://github.com/AzuraCast/AzuraCast/discussions/6386)]

for the webpage i use the API, with the API the track is shown wel, i created a nowplaying.php :

![NOWPLAYING](https://github.com/AzuraCast/AzuraCast/assets/62739604/7176ddaa-c10c-4cdd-9f56-7ff67b1dda4d)


Under the Player is running the nowplaying.php (Refreshes every 30 seconds), have a look here https://piratenkanon.fm/flac/
